src/browser/web-preview.js

```diff
@@ -223,26 +225,40 @@ define((require, exports, module) => {
   exports.viewPreview = viewPreview;
 
   const style = StyleSheet.create({
-    preview: {
-      width: '100vw',
-      height: '100vh',
-      paddingTop: 'calc(100vh / 2 - 150px)',
+    scroller: {
       backgroundColor: '#273340',
-      overflowX: 'auto',
+      height: '100vh',
+      width: '100vw',
+      overflowY: 'hidden',
       position: 'absolute',
       top: 0,
       zIndex: 0,
-      MozWindowDragging: "drag"
+      MozWindowDragging: 'drag',
+    },
+    previews: {
+      // This is important. We need previews to make space around itself.
+      // Margin doesn't play well with scroll -- the right-hand edge will get
+      // cut off, so we turn on the traditional CSS box model and use padding.
```

Comment:
Margin has always had problems interacting with scrollable wrappers, in all browsers.

We use `float:left` because `display:inline-block` renders additional gaps in between elements when space or carriage returns are present in the source code. Anyway, cards are a fixed height, so we can just fix the container height.